jenkins/opensearch-dashboards/integ-test.jenkinsfile

```diff
@@ -0,0 +1,137 @@
+lib = library(identifier: "jenkins@20211118", retriever: legacySCM(scm))
+
+pipeline {
+    options {
+        timeout(time: 3, unit: 'HOURS')
```

Comment:
Does it take 3 hours to run this job?

Reply:
As of right now no, but I do not have metrics to determine the time the job will take once a full test suite of plugins onboarded to the new integration test repo with security and without. So opted for the a long time now and then once it gets running I think it can be revisited. Most likely being capped at 2 hours  (from the actual run time of the integration tests when I run them locally)